README.md

```diff
@@ -327,17 +327,17 @@ natural algebraic approach.
 using JuMP, Gurobi
 
 # pass params as keyword arguments to GurobiSolver
-m = Model(solver=GurobiSolver(Presolve=0))
+model = Model(with_optimizer(Gurobi.Optimizer, Presolve=0))
 
-@variable(m, x >= 5)
-@variable(m, y >= 45)
+@variable(model, x >= 5)
+@variable(model, y >= 45)
 
-@objective(m, Min, x + y)
-@constraint(m, 50x + 24y <= 2400)
-@constraint(m, 30x + 33y <= 2100)
+@objective(model, Min, x + y)
+@constraint(model, 50x + 24y <= 2400)
+@constraint(model, 30x + 33y <= 2100)
 
-status = solve(m)
-println("Optimal objective: ",getobjectivevalue(m),
+status = solve(model)
+println("Optimal objective: ",getobjectivevalue(model),
 	". x = ", getvalue(x), " y = ", getvalue(y))
```

Comment:
`getobjectivevalue` -> `objective_value` and `getvalue` -> `value`.

Reply:
Fixed.